t/deploy.t

```diff
@@ -48,7 +48,7 @@ my $dh = DBIx::Class::DeploymentHandler->new(
 my $deployed_version;
 try {
     $deployed_version = $dh->version_storage->database_version;
-};
+}
```

Comment:
that's invalid syntax and needs to be

```suggestion
}
catch ($e) { }
```

However as this is part of tests we should use functions from Test::Exception